duckchat/duckchat-impl/src/main/java/com/duckduckgo/duckchat/impl/inputscreen/ui/InputScreenConfigResolver.kt

```diff
@@ -42,7 +39,7 @@ interface InputScreenConfigResolver {
 @SingleInstanceIn(scope = ActivityScope::class)
 class InputScreenConfigResolverImpl @Inject constructor(
     private val duckChatInternal: DuckChatInternal,
-    @ActivityContext private val activityContext: Context,
+    private val appCompatActivity: AppCompatActivity,
```

Comment:
I noticed that we can inject the whole activity instead of just the context, which saves us passing around the intent and prevents any race conditions.

It required updating unit tests though 🫣